packages/yourloops/lib/data/api.ts

```diff
@@ -33,7 +33,7 @@ import _ from "lodash";
 import { PatientData } from "models/device-data";
 import { MessageNote, MessagesThread } from "models/message";
 import { HttpHeaderKeys, HttpHeaderValues } from "../../models/api";
-import { User, UserRoles } from "../../models/shoreline";
+import { User, UserRoles, ComputedTIR } from "../../models/shoreline";
```

Comment:
the computedTIR should be part of shoreline. I think we should create a folder medicaldata (or maybe just data) cointining these declarations.

Reply:
shouldn't ?